apps/src/templates/EndOfLessonDialog.jsx

```diff
@@ -29,7 +29,7 @@ function EndOfLessonDialog({lessonNumber, isSummaryView}) {
       handleClose={handleClose}
       style={styles.dialog}
     >
-      <h2 id="uitest-end-of-lesson-header">
+      <h2 className="uitest-end-of-lesson-header">
```

Comment:
Just curious, why did you change this from an id to a class?

Reply:
I noticed in other places we were using classes for the uitest tag